requirements.txt

```diff
@@ -1 +1,2 @@
 openfermion>=0.1
+pyscf>=1.4
```

Comment:
if we add this to requirements then pypi will try to install automatically. I worry that installing pyscf automatically from pypi has had some problems in the past. Let's leave this out of this PR and start a new PR where we try to setup travis. If it passes on travis, then we can add this here.